ports/esp32/main.c

```diff
@@ -63,6 +63,15 @@
 #if MICROPY_PY_ESPNOW
 #include "modespnow.h"
 #endif
+#if MBEDTLS_VERSION_NUMBER >= 0x03000000
+#include "mbedtls/build_info.h"
+#include "mbedtls/platform_time.h"
+#else
+#include "mbedtls/version.h"
+#endif
+#ifdef MICROPY_MBEDTLS_PLATFORM_TIME_ALT
+#include "mbedtls/mbedtls_config.h"
```

Comment:
I think you want to include `#include "mbedtls/platform.h"` instead.

Reply:
`mbedtls/platform.h` includes `mbedtls/platform_time.h`, since this only requires `mbedtls_platform_set_time` I think it's simpler to include `platform_time` directly.